[2025-04-06T18:51:26.021+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: great_expectations_validation.check_table_exists manual__2025-04-06T18:51:21.234125+00:00 [queued]>
[2025-04-06T18:51:26.034+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: great_expectations_validation.check_table_exists manual__2025-04-06T18:51:21.234125+00:00 [queued]>
[2025-04-06T18:51:26.035+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 4
[2025-04-06T18:51:26.055+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): check_table_exists> on 2025-04-06 18:51:21.234125+00:00
[2025-04-06T18:51:26.066+0000] {standard_task_runner.py:57} INFO - Started process 4684 to run task
[2025-04-06T18:51:26.073+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'great_expectations_validation', 'check_table_exists', 'manual__2025-04-06T18:51:21.234125+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/data_validation_dag.py', '--cfg-path', '/tmp/tmp19xu5xjl']
[2025-04-06T18:51:26.074+0000] {standard_task_runner.py:85} INFO - Job 22: Subtask check_table_exists
[2025-04-06T18:51:26.161+0000] {task_command.py:410} INFO - Running <TaskInstance: great_expectations_validation.check_table_exists manual__2025-04-06T18:51:21.234125+00:00 [running]> on host 6f966d9d4c8d
[2025-04-06T18:51:26.306+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='great_expectations_validation' AIRFLOW_CTX_TASK_ID='check_table_exists' AIRFLOW_CTX_EXECUTION_DATE='2025-04-06T18:51:21.234125+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-06T18:51:21.234125+00:00'
[2025-04-06T18:51:26.308+0000] {logging_mixin.py:150} INFO - Connexion string vérifiée: postgresql://sportadvantagehr:sportpassword@business-postgres:5432/sportadvantages
[2025-04-06T18:51:26.309+0000] {logging_mixin.py:150} INFO - Paramètres de connexion: host=business-postgres, port=5432, user=sportadvantagehr, database=sportadvantages
[2025-04-06T18:51:26.310+0000] {logging_mixin.py:150} INFO - Tentative de connexion 1/5
[2025-04-06T18:51:26.320+0000] {logging_mixin.py:150} INFO - Connexion réussie!
[2025-04-06T18:51:26.324+0000] {logging_mixin.py:150} INFO - Exécution de la requête: SELECT schema_name FROM information_schema.schemata;
[2025-04-06T18:51:26.329+0000] {logging_mixin.py:150} INFO - Tous les schémas disponibles: [('pg_toast',), ('pg_catalog',), ('public',), ('information_schema',), ('sport_advantages',), ('pg_temp_4',), ('pg_toast_temp_4',)]
[2025-04-06T18:51:26.330+0000] {logging_mixin.py:150} INFO - Exécution de la requête: SELECT schema_name FROM information_schema.schemata WHERE schema_name = 'sport_advantages';
[2025-04-06T18:51:26.331+0000] {logging_mixin.py:150} INFO - Schéma sport_advantages: [('sport_advantages',)]
[2025-04-06T18:51:26.332+0000] {logging_mixin.py:150} INFO - Exécution de la requête: SELECT table_name FROM information_schema.tables WHERE table_schema = 'sport_advantages';
[2025-04-06T18:51:26.338+0000] {logging_mixin.py:150} INFO - Toutes les tables dans le schéma sport_advantages: [('employees',), ('commute_validations',), ('sport_activities',), ('sport_advantages',), ('employees_masked',)]
[2025-04-06T18:51:26.339+0000] {logging_mixin.py:150} INFO - Exécution de la requête: 
                SELECT table_name 
                FROM information_schema.tables 
                WHERE table_schema = 'sport_advantages' 
                AND table_name = 'sport_activities';
                
[2025-04-06T18:51:26.341+0000] {logging_mixin.py:150} INFO - Table sport_activities: [('sport_activities',)]
[2025-04-06T18:51:26.341+0000] {logging_mixin.py:150} INFO - ✅ La table 'sport_advantages.sport_activities' existe!
[2025-04-06T18:51:26.342+0000] {logging_mixin.py:150} INFO - Exécution de la requête: SELECT * FROM sport_advantages.sport_activities LIMIT 5;
[2025-04-06T18:51:26.344+0000] {logging_mixin.py:150} INFO - Échantillon de données: [(1, 70643, datetime.datetime(2024, 4, 5, 11, 0, 3), 'Équitation', None, 3918, None), (2, 34382, datetime.datetime(2024, 4, 5, 14, 45, 38), 'Musculation', None, 5035, None), (3, 50316, datetime.datetime(2024, 4, 5, 14, 40, 45), 'Judo', None, 3254, None), (4, 19773, datetime.datetime(2024, 4, 6, 19, 19, 24), 'Corde à sauter', None, 5696, None), (5, 99401, datetime.datetime(2024, 4, 6, 10, 28, 43), 'Randonnée', 6770, 6449, None)]
[2025-04-06T18:51:26.345+0000] {logging_mixin.py:150} INFO - Connexion fermée
[2025-04-06T18:51:26.346+0000] {python.py:183} INFO - Done. Returned value was: True
[2025-04-06T18:51:26.386+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=great_expectations_validation, task_id=check_table_exists, execution_date=20250406T185121, start_date=20250406T185126, end_date=20250406T185126
[2025-04-06T18:51:26.448+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2025-04-06T18:51:26.485+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
